voucher/urls.py

```diff
@@ -3,8 +3,11 @@
 
 # SharedAPIRootRouter is automatically imported in global urls config
 router = SharedAPIRootRouter()
-router.register(r'voucher/wallets', WalletViewSet, base_name='voucher_wallets')
+router.register(r'voucher/wallets/voucher', VoucherWalletViewSet, base_name='voucher_wallets')
+router.register(r'voucher/wallets/coffee', CoffeeWalletViewSet, base_name='coffee_wallets')
+router.register(r'voucher/registerlogs/voucher', WorkLogViewSet)
+router.register(r'voucher/registerlogs/coffee', RegisterLogViewSet)
+router.register(r'voucher/uselogs/voucher', VoucherUseLogViewSet)
+router.register(r'voucher/uselogs/coffee', CoffeeUseLogViewSet)
```

Comment:
Hm. I og for seg enig i mye av det du sier her. Jeg bare tenker at vanlige bonger og kaffebonger i og for seg er mye av det samme. Er det en mulighet å lage dummy-brukere som kan binde NfcCard og Wallet sammen? Da kan man også koble sammen flere kort til en wallet, og peke den på riktig bruker hvis man ønsker det